### PR TITLE
Retrieve reporting translations + tests

### PR DESCRIPTION
**Pull request by ahemphill**

### CODE REVIEW

File: tools/scoring/score/parse_config.py [hunk 1]
```diff
@@ -125,3 +125,37 @@ def match_reporting_entities(
           matches.append(proposed_entity.cloud_device_id)
 
     return matches
+
+  @staticmethod
+  def retrieve_reporting_translations(
+      *, matches: List[str], proposed: Dict[str, EntityInstance],
```
Comment (charbull) [inline tools/scoring/score/parse_config.py:131]: can you please detail what is the input in the docstring.
matches is a list of cloud device id for matched entities.
proposed ...

File: tools/scoring/score/parse_config.py [hunk 2]
```diff
@@ -125,3 +125,37 @@ def match_reporting_entities(
...
+        A dictionary with `cloud_device_id`s as keys
+        and lists of translation tuples as values
+    """
+
+    translations = {}
+    for cloud_device_id in matches:
+      find_entity = lambda dictionary, cdid=cloud_device_id: [
+          entity for entity in dictionary.values()
+          if entity.cloud_device_id == cdid
+      ][0]
```
Comment (charbull) [inline tools/scoring/score/parse_config.py:147]: this is neat

Comment (charbull) [reply tools/scoring/score/parse_config.py:147]: can you add some comments please

File: tools/scoring/score/parse_config.py [hunk 3]
```diff
@@ -125,3 +125,37 @@ def match_reporting_entities(
...
+
+      proposed_entity = find_entity(proposed)
```
Comment (charbull) [inline tools/scoring/score/parse_config.py:149]: same here what are we retrieving?

**Review by charbull — COMMENTED** [review]
just more comments please :)

**Comment by ahemphill** [comment]
https://github.com/google/digitalbuildings/pull/465/commits/aea86aaa05be7b8a1b78f64d602c23cb12e0c20d

**Review by charbull — APPROVED** [review]

**Pull request merged**